src/test/java/com/robin/hoteless/service/UserServiceTest.java

```diff
@@ -0,0 +1,30 @@
+package com.robin.hoteless.service;
+
+import com.robin.hoteless.model.User;
+import com.robin.hoteless.repository.UserRepository;
+import junit.framework.TestCase;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+@Service
+public class UserServiceTest extends TestCase {
+    @Autowired
+    UserRepository userRepository;
+
+    @Test
+    public void testGetAll() {
+        // I need to better understand the correct way to test this
+    }
+
+    @Test
+    public void testSaveUser() {
+        // userRepository is null here - not sure if I was going about testing this the right way
```

Comment:
That is because the UserRepository isn't set up in a new @Bean config item, so @Autowiring doesn't find any @Beans to implement here.